### DISCUSSION TITLE
Deletion of photos once uploaded to our platform

### DISCUSSION THREAD

**Discussion by DiegoOrtiz-npc** [immich-app/immich#9032] [General]
First of all, I'm new here and I really think the project is great.

One feature that I see is missing is that once you have uploaded the photo to the platform, it (through configuration in case someone does not want it) is capable of automatically deleting it from the device once you have uploaded the photo.

This is currently done by Google Photos and right now I am looking for a third-party application that will do it for me.
If this were already integrated there would be no need to do it.

Just for this feature, I'm looking at whether or not to use the software.

NOTE: very good project.

**Comment by bo0tzz** [comment]
#165
